### PR TITLE
CCE-383: Grouped cards 

### PR DESCRIPTION
**Pull request by joseegm**

### CODE REVIEW

File: src/less/techne-patterns/_data-card.less [hunk 1]
```diff
@@ -93,6 +171,42 @@
     flex-direction: column;
   }
 
+  .cardRow {
+    border-top: 1px solid transparent;
+    &.cardRowClickable {
+      &:hover, &:focus {
```
Comment (saadmhybris) [inline src/less/techne-patterns/_data-card.less]: please put &:focus on next line. sorry nitpicky :)

File: src/less/techne-patterns/_data-card.less [hunk 2]
```diff
@@ -93,6 +171,42 @@
...
+  &.dataCards-grouped{
+    margin-bottom: 20px;
+    
+    .dataCard{
+      margin-bottom: 0px;
+      margin-right: 0px;
+      border-top: none !important;
+      border-right: 1px solid rgba(138, 143, 161, 0.3);
+      border-left: 1px solid rgba(138, 143, 161, 0.3);
+      box-shadow: 0 0px 0 rgba(138,143,161,.3);
```
Comment (saadmhybris) [inline src/less/techne-patterns/_data-card.less]: maybe color vars here instead of rgba?

File: src/less/techne-patterns/_data-card.less [hunk 3]
```diff
@@ -93,6 +171,42 @@
...
+      
+      .cardRow{
+        &.cardRowClickable{
+          border: 1px solid transparent;
+
+          &:hover,
+          &:focus {
+            border-top: 1px solid rgba(138, 143, 161, 0.3);
```
Comment (saadmhybris) [inline src/less/techne-patterns/_data-card.less]: var of color?

File: src/less/globals/themes/_techne.less [hunk 4]
```diff
@@ -887,6 +887,14 @@
...
 @panel-danger-heading-bg:     @default-greyblue-100;
 
+//== Cards
+//
+//##
+
+// Border color
+@cards-border-color:        rgba(138, 143, 161, 0.3);
+@cards-shadowcolor:         rgba(138, 143, 161, 0.3);
+
```
Comment (joseegm) [inline src/less/globals/themes/_techne.less]: Changed, was a typo, on the usage is correct, dunno how it that worked O_o

**Pull request merged**